### DISCUSSION TITLE
Rename env var `TORRUST_TRACKER_DATABASE` to `TORRUST_TRACKER_DATABASE_DRIVER`

### DISCUSSION THREAD

**Discussion by josecelano** [torrust/torrust-tracker#415] [Proposals]
Hi @da2ce7, I think the name `TORRUST_TRACKER_DATABASE` can be confusing. I thought it was the database name. In the tracker configuration file (and in the code), we use the word `driver` to refer to the type of database system the tracker uses.

In fact, that could be the prefix (`TORRUST_TRACKER_DATABASE_*`) for all options related to the database.

```
db_driver = "Sqlite3"
db_path = "./storage/database/data.db"
```

**Comment by da2ce7** [comment]
Yep, `TORRUST_TRACKER_DATABASE_DRIVER` is a better naming.

**Comment by da2ce7** [comment]
done in #418
